battle-service/src/main/java/com/company/battle/services/SettingsService.java

```diff
@@ -0,0 +1,23 @@
+package com.company.battle.services;
+
+import com.company.common.dtos.SettingsDto;
+import com.company.common.models.SettingsEntity;
+import org.springframework.stereotype.Service;
+
+import java.util.UUID;
+
+@Service
+public class SettingsService {
```

Comment:
You don't need this due to this is part of game and settings can't be used without game context